### DISCUSSION TITLE
iOS app forwards to browser when viewing compare links

### DISCUSSION THREAD

**Discussion by LogicalJoe** [community/community#42895] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

I usually use the iOS app for reviews and issues, but whenever I have a compare link, i.e. https://github.com/hexfiend/hexfiend/compare/a12acab9e94e...b721e947a325, the app just forwards to the default browser, which is rather frustrating for private repositories where I would then need to log in to view it the content.  Is this configurable or just not supported (yet)?  A way to view these in the app would be nice.

**Comment by HenSquared** [comment, marked as answer]
Hey @LogicalJoe , thanks for the feedback. That can definitely be frustrating and would be a lot better to be able to compare more easily using compare links like these. 

I've opened an issue internally to discuss with the team on how we can support comparing two commits more easily like this.
